src/main/java/Task.java

```diff
@@ -0,0 +1,39 @@
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+import java.time.temporal.ChronoUnit;
+
+public class Task {
+    protected String description;
+    protected boolean isDone;
+    protected LocalDate date;
+    protected String duration;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public Task(String description, LocalDate date) {
+        this.description = description;
+        this.date = date;
+    }
+
+    public Task(String description, LocalDate date, String duration) {
+        this.description = description;
+        this.date = date;
+        this.duration = duration;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "\u2713" : "\u2718"); //return tick or X symbols
+    }
```

Comment:
I think it's great that your UI text will have user friendly icons! However, I do suggest that since these are Magical Constants, perhaps they can be defines as top level class members as private and final using the appropriate Caps Lock separation instead of placing them in the middle of your code as this may confuse future reviewers!